test/mock-kuzzle.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const mockRequire = require('mock-require');
+
+function mockKuzzle () {
+  const KuzzleMock = require('./mocks/kuzzle.mock');
+  mockRequire('../lib/kuzzle/kuzzle', KuzzleMock);
+}
+
+// BAD but, oh well, it makes things so much easier...
+const savedStopAll = mockRequire.stopAll.bind(mockRequire);
+
+mockRequire.stopAll = () => {
+  savedStopAll();
+  mockKuzzle();
+};
+
+module.exports = mockKuzzle;
+
+mockKuzzle();
+
```

Comment:
This looks like a leftover from my first attempt to globalize the kuzzle object. This file has never been pushed to 2-dev since we now use `global.kuzzle` :thinking: